Bender.lock

```diff
@@ -1,7 +1,7 @@
 packages:
   common_cells:
-    revision: 9afda9abb565971649c2aa0985639c096f351171
-    version: 1.38.0
+    revision: de8742553f36321af3997418cb632907081c273a
+    version: null
```

Comment:
Let's not use a custom common_cells version on the main branch here - please use a proper version tag.